template_feed/Microsoft.DotNet.Common.ItemTemplates/content/RazorPageWithPageModel/Index.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
```

Comment:
This is the same set of namespaces that the template uses for `HomeController` with the `Microsoft.AspNetCore.Mvc.RazorPages` namespace added (required).